test/runtests.sh

```diff
@@ -0,0 +1,4 @@
+set -e
```

Comment:
i forget the reason, but I've always used (and been told to use) set instead of the shebang line.